build/scripts/generate_latest_metas.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
```

Comment:
Wrong copyright date.

Also we should probably have a .gitignore for /latest/ folders so that we don't commit them to the repo, as they'll be generated from this point onward.

Reply:
Good catch, will fix

Reply:
Done